cmd/lotus-miner/sectors.go

```diff
@@ -85,12 +86,23 @@ var sectorsStatusCmd = &cli.Command{
 	ArgsUsage: "<sectorNum>",
 	Flags: []cli.Flag{
 		&cli.BoolFlag{
-			Name:  "log",
-			Usage: "display event log",
+			Name:    "log",
+			Usage:   "display event log",
+			Aliases: []string{"l"},
 		},
 		&cli.BoolFlag{
-			Name:  "on-chain-info",
-			Usage: "show sector on chain info",
+			Name:    "on-chain-info",
+			Usage:   "show sector on chain info",
+			Aliases: []string{"c"},
+		},
+		&cli.BoolFlag{
+			Name:    "partition-info",
+			Usage:   "show partition related info",
+			Aliases: []string{"p"},
+		},
+		&cli.BoolFlag{
+			Name:  "proof",
```

Comment:
Since I'm probably not alone in not knowing the `usage` field would benefit from explaining this: "print proof of replication for sector", or "print latest proof of spacetime for sector"